api/db-cdr/generate-cdr/cb-criteria-parallel/debug/comp-tables.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+BQ_PROJECT=$1        # project - 'all-of-us-ehr-dev'
+BQ_DATASET=$2        # dataset - 'BillDummyMult'
+BQ_ORIG_DATASET= $3  #'Chenchal_R2021Q3R2_v2'
+
+crit_table=cb_criteria_sorted
+#orig_crit_table=cb_criteria_original
```

Comment:
Can this line go away since it's commented out?

Reply:
done